docs/state.md

```diff
@@ -34,7 +34,8 @@ AppStore
       }
     }
   },
-  sites: [{
+  sites: {
+  [siteKey]: { // Calculated by siteUtil.getSiteKey()
```

Comment:
could you just put the format of the key? like folder specifies folderId, URLs use location/partition/folderId?